app/reflexes/admin/orders_reflex.rb

```diff
@@ -134,4 +134,8 @@ def render_business_number_required_error(distributors)
       morph_admin_flashes
     end
   end
+
+  module Constants
+    PATHS = %w[edit customer payments adjustments invoices return_authorizations].freeze
+  end
```

Comment:
I don't think that you need an additional module just to declare a constant.

Reply:
I agree with you, but there is a 100 lines limit for Class size, so I had to do this not to raise the rubocop offense.

Reply:
Oh, this is a good example for style rules producing worse code. Rubocop is supposed to help us. If its recommendation is stupid, we need to do something else.

I agree with Rubocop that the class is getting too big. Adding a module doesn't help though. We could move some code into other files like an InvoiceReflex, for example. But since we are going to shift away from reflexes, we can just add it to the rubocop todo file.

Reply:
So, I added a new line on the rubocop todo file :)